scripts/cli.ts

```
@@ -89,14 +101,74 @@ async function checkUpdates() {
+		expected.add(name);
+	}
+
+	return expected;
+}
+
+function removeSubmodule(submodulePath: string): void {
+	// De-initialize the submodule
+	// cspell:ignore deinit
+	execSafe(`git submodule deinit -f ${submodulePath}`);
+	// Remove from .git/modules
+	const gitModulesPath = join(ROOT, ".git", "modules", submodulePath);
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)